tests/integration/relation_tests/test_charm_relations.py

```diff
@@ -34,8 +35,13 @@
 ANOTHER_DATABASE_APP_NAME = "another-database"
 APP_NAMES = [APPLICATION_APP_NAME, DATABASE_APP_NAME, ANOTHER_DATABASE_APP_NAME]
 TEST_APP_CHARM_PATH = "tests/integration/relation_tests/application-charm"
+REQUIRED_UNITS = 2
 
 
+@pytest.mark.skipif(
+    os.environ.get("PYTEST_SKIP_DEPLOY", False),
+    reason="skipping deploy, model expected to be provided.",
+)
```

Comment:
remove this for the same reasons that we removed the other one

Reply:
done, thank you